tests/store/crud/test_update.py

```diff
@@ -58,7 +58,7 @@ def test_update_analysis_jobs_from_slurm_jobs(analysis_store: MockStore, squeue_
         analysis=analysis, squeue_result=squeue_result
     )
     updated_analysis: Analysis = analysis_store.get_analysis(
-        case_id=analysis.family, started_at=analysis.started_at, status=TrailblazerStatus.PENDING
+        case=analysis.family, started_at=analysis.started_at, status=TrailblazerStatus.PENDING
```

Comment:
I would go here for `case_name` or `case_internal_id`, as case seems like a Family object like in cg

Reply:
Using case_name as there is no internal case id in Traiblazer so it would not make sense and couple to cg too tigthly